types/node-zendesk/index.d.ts

```diff
@@ -90,6 +90,7 @@ export namespace Attachments {
             fileOptions: {
```

Comment:
(can't comment outside of ranges on GitHub)

I think `file` should be a `PathLike | Buffer | stream` on here and line 98 (where stream is from `node:stream`)

Technically `PathLike` will already allow a Buffer but this is misleading: a Buffer in a `PathLike` would be expected to be converted to a string to find a file path. So making the `| Buffer` explicit is clearer I think.

`stream` is based on https://github.com/blakmatrix/node-zendesk/blob/master/lib/client/client.js#L220

Reply:
That makes sense to be explicit about that :thumbsup: good call!